posture/serialnumber_notmacos_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Build on Windows, Linux and *BSD
+
+//go:build !darwin && !android && !js && !plan9 && !solaris
```

Comment:
Let's match the other file?

```suggestion
//go:build windows || (linux && !android) || freebsd || openbsd || dragonfly || netbsd
```